README.md

```diff
@@ -12,3 +12,8 @@ Bot that will send telegram message in case of electricity state was changed.
 6. Bot is ready, run it via `poetry run electricitybot` command or via `make run` command
 
 Bot will ping provided IP adress specified in `IP_TO_CHECK` variable. In case if there will be no response for 4 ping requests in a row, bot will send message about power outage and vice versa.
+
+## How to use in Docker
+1. build docker image with `docker build . -t yurnov/electricitybot`
```

Comment:
yurnov/electricitybot its a part of your fork

Reply:
I will rename it to f1ashhimself/electricitybot, is it ok?

Reply:
Sure